samples/camera/README.rst

```diff
@@ -0,0 +1,28 @@
+.. _camera-sample:
+
+Camera Application
+##################
+
+Overview
+********
+
+This camera application is actually a port of the 4D Systems uCAM-III camera
+module in Zephyr. It defines its own API.
+
+Requirements
+************
+
+The demo assumes that a uCAM-III is connected to the UART_2 TX/RX of a Nucleo
+L496ZG. Also, the camera module should be powered by the 5V pin.
+
+Building and Running
+********************
+
+This sample outputs the process of interfacing with the camera module to the
```

Comment:
I'm not sure what "outputs the process of interfacing with the camera module" means?  Does it write some debugging output to the console?  (If so can you show an example here.) Does it only produce still images or a video stream? Is there a JPEG image captured somewhere?  What is the API presented by the camera module (link to that would be good)?  Is the whole API implemented in this port?